Resources/doc/features.md

```diff
@@ -0,0 +1,12 @@
+Features
+===========
+
+Supports multiple entity managers 
+-------------
+
+Configure multiple entity managers according to http://symfony.com/doc/current/cookbook/doctrine/multiple_entity_managers.html
```

Comment:
This should also be moved to setup. It's a specific way of setting up the EM so that the ERB can intelligently handle them.